.github/workflows/go-licenses.yml

```diff
@@ -0,0 +1,62 @@
+name: go-licenses
+
+on:
+  # run action when a change lands in the main branch which updates go.mod or
+  # our license template file. Also allow manual triggering.
+  push:
+    branches:
+      - main
+    paths:
+      - go.mod
+      - .github/licenses.tmpl
+      - .github/workflows/go-licenses.yml
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-$${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  tailscale:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version-file: go.mod
+
+      - name: Install go-licenses
+        run: |
+          go install github.com/google/go-licenses@v1.2.2-0.20220825154955-5eedde1c6584
+
+      - name: Run go-licenses
+        env:
+          # include all build tags to include platform-specific dependencies
+          GOFLAGS: "-tags=cgo,darwin,freebsd,ios,js,linux,openbsd,wasm,windows"
+        run: |
+          mkdir -p licenses
+          go-licenses report tailscale.com/cmd/tailscale tailscale.com/cmd/tailscaled > licenses/tailscale.md --template .github/licenses.tmpl
+
+      - name: Get access token
+        uses: tibdex/github-app-token@f717b5ecd4534d3c4df4ce9b5c1c2214f0f7cd06 # v1.6.0
```

Comment:
You reviewed the contents of this action's repo and the one below at this revision? Admittedly passing our access tokens to third-party code makes me nervous, given that this one has the power to send apparently-genuine PRs.

Reply:
yep, that is the biggest concern with this action (and why I wanted your review), and something I talked through with @DentonGentry yesterday.  

Using a GitHub App gives us [pretty granular permissions](https://github.com/peter-evans/create-pull-request/blob/main/docs/concepts-guidelines.md#authenticating-with-github-app-generated-tokens)... the app is only installed on this repo, and branch protection ensures it can't touch the main branch.  I've done a basic review of the code of both actions, and they seem to do only what they say on the tin.  I'm also pinning at specific versions of everything so the code can't change underneath us.

Reply:
hmmm... I just checked again and it appears this commit actually [only includes the compiled version](https://github.com/tibdex/github-app-token/tree/f717b5ecd4534d3c4df4ce9b5c1c2214f0f7cd06) of the code.  I was actually reviewing at main.  :-\